v22.2/ui-transactions-page.md

```diff
@@ -7,22 +7,65 @@ docs_area: reference.db_console
 
 {{site.data.alerts.callout_success}}
-In contrast to the [**Statements** page](ui-statements-page.html), which displays [SQL statement fingerprints](ui-statements-page.html#sql-statement-fingerprints), the **Transactions** page displays SQL statement fingerprints grouped by [transaction](transactions.html).
+In contrast to the [**Statements** page](ui-statements-page.html), which displays [SQL statement fingerprints](ui-statements-page.html#sql-statement-fingerprints), the **Transactions** page displays transaction fingerprints, that is, SQL statement fingerprints grouped by [transaction](transactions.html).
 {{site.data.alerts.end}}
 
+## Transaction Fingerprints view
+
+The **Transaction Fingerprints** view helps you:
+
+- Identify frequently [retried]({{ link_prefix }}transactions.html#transaction-retries) transactions.
+- [Troubleshoot]({{ link_prefix }}query-behavior-troubleshooting.html) high latency transactions or execution failures.
+- View transaction [details](#transaction-details-page).
+
 {% if page.cloud != true %}
-To view this page, click **SQL Activity** in the left-hand navigation of the DB Console. Click the **Transactions** tab.
+To view this page, click **SQL Activity** in the left-hand navigation of the DB Console.
 {% else %}
 To view this page, click **SQL Activity** in the left-hand navigation of the {{ site.data.products.db }} Console. Click the **Transactions** tab.
 {% endif %}
 
+Click the **Transactions** tab. The **Transaction Fingerprints** radio button is selected and the [Transactions table](#transactions-table) displays.
+
+The following screenshot shows the transaction fingerprint for `SELECT city, id FROM vehicles WHERE city = $1` while running the [`movr` workload](cockroach-workload.html#run-the-movr-workload).
+
+<img src="{{ 'images/v22.2/transaction-fingerprint.png' | relative_url }}" alt="Transaction fingerprint" style="border:1px solid #eee;max-width:100%" />
+
+If you click the transaction fingerprint in the **Transactions** column, the [**Transaction Details** page](#transaction-details-page) displays.
+
+<img src="{{ 'images/v22.2/transaction-details.png' | relative_url }}" alt="Transaction details" style="border:1px solid #eee;max-width:100%" />
+
+## Active Executions view
+
+The **Active Executions** view helps you:
+
+- Understand and tune workload performance, particularly long-running transactions.
+
+{% if page.cloud != true %}
+To display this view, click **SQL Activity** in the left-hand navigation of the DB Console.
+{% else %}
+To display this view, click **SQL Activity** in the left-hand navigation of the {{ site.data.products.db }} Console.
+{% endif %}
+
+The **Statements** tab is selected. Click the **Transactions** tab and the **Active Executions** radio button. The [Active Executions table](#active-executions-table) displays.
```

Comment:
```suggestion
When the **Transactions** tab is selected, click the **Active Executions** radio button. The [Active Executions table](#active-executions-table) displays.
```

Reply:
When you click SQL Activity, the default is the Statements tab and the * Fingerprints radio button.  You need to first click the Transactions tab. So the original is accurate.

Reply:
See previous comments about this UI behavior.